yggdrasil.go

```diff
@@ -107,6 +110,19 @@ func main() {
 		if err != nil {
 			panic(err)
 		}
+		// If there's a byte order mark - which Windows 10 is now incredibly fond of
+		// throwing everywhere when it's converting things into UTF-16 for the hell
+		// of it - remove it and decode back down into UTF-8. This is necessary
+		// because hjson doesn't know what to do with UTF-16 and will panic
+		if bytes.Compare(config[0:2], []byte{0xFF, 0xFE}) == 0 ||
+		  bytes.Compare(config[0:2], []byte{0xFF, 0xFF}) == 0 {
```

Comment:
shouldn't it be `[]byte{0xFE, 0xFF}`?
https://en.wikipedia.org/wiki/Byte_order_mark#Byte_order_marks_by_encoding

Reply:
You are indeed correct - that's a typo. I'll push a fix to the `develop` branch for merging into the next release.